mlperf_common/mountcheck.py

```diff
@@ -0,0 +1,95 @@
+import sys
+import os
+import json
+import subprocess
+
+def du(path):
+    return subprocess.check_output(['du','-sk', path]).split()[0].decode('utf-8')
+
+def getMountInfo(paths_to_verify):
+
+    paths_to_verify = paths_to_verify.split(",")
+
+    for dir_path in paths_to_verify:
+        if not os.path.exists(dir_path):
+            print("Error: path $dir_path is incorrect")
+            sys.exit(1)
+
+    cont_mount_info = []
+    for dir_path in paths_to_verify:
+        records = []
```

Comment:
recommend breaking this loop body (lines 20-33) out into its own procedure taking a single dir_path as input and returning a `records` datastructure.

Reply:
fixed